pkg/sql/logictest/testdata/logic_test/pg_catalog

```diff
@@ -383,6 +383,43 @@ v1            a        false         true        0            NULL    NULL
 v1            b        false         true        0            NULL    NULL        NULL
 v1            c        false         true        0            NULL    NULL        NULL
 
+# Check relkind codes.
+statement ok
+CREATE DATABASE relkinds
+
+statement ok
+SET DATABASE = relkinds
+
+statement ok
+CREATE TABLE tbl_test (k int primary key, v int)
+
+statement ok
+CREATE INDEX tbl_test_v_idx ON tbl_test (v)
+
+statement ok
+CREATE VIEW view_test AS SELECT k, v FROM tbl_test ORDER BY v
+
+statement ok
+CREATE SEQUENCE seq_test
+
+query TT
+SELECT relname, relkind
+FROM pg_catalog.pg_class c
+JOIN pg_catalog.pg_namespace n ON c.relnamespace = n.oid
+WHERE n.nspname = 'relkinds'
+ORDER BY relname
```

Comment:
stuff like this really makes me wish we had some existing way in a logic test to prove that this actually matched Postgres. It's been talked about having logic tests that run against both but it's not immediately clear to me how to make that work (I did manually check this one).

Reply:
Yeah that'd be great. Pretty annoying to have to manually test these. (Thx for doing it)